package.json

```diff
@@ -43,21 +43,21 @@
     "@babel/preset-env": "^7.9.6",
     "@babel/preset-flow": "^7.9.0",
     "cross-env": "^7.0.2",
-    "eslint": "^6.8.0",
-    "eslint-config-bliss": "^4.8.1",
+    "eslint": "^7.0.0",
```

Comment:
Should the `peerDependencies` entry also get `|| ^7.0.0`?

Reply:
Ah good catch, I'll update those as well